### DISCUSSION TITLE
Getting error with publishing workflow on 2.9.2

### DISCUSSION THREAD

**Discussion by woola168** [elsa-workflows/elsa-core#3751] [Q&A]
I'm getting error code 500 when publishing workflow
and the error was
Microsoft.EntityFrameworkCore.Database.Command: Error: Failed executing DbCommand (0ms) [Parameters=[@p0='?' (Size = 4000)], CommandType='Text', CommandTimeout='30']
DELETE FROM "Triggers" WHERE `WorkflowDefinitionId` = @p0

I was using mysql as my database

**Comment by sfmskywalker** [comment]
Does this happen all the time, did it start happening recently l. Do you have steps to reproduce this in a new clean solution?
